doc/manuals/admin/sanity_check.md

```diff
@@ -24,14 +24,15 @@ curl --header 'Accept: application/json' localhost:1026/version
 ```
 {
```

Comment:
Changes in .md in this PR are very somehow aesthetic, but an improvement after all.

I have also covered the Japanese translation files, given that, although I'm afraid I don't know Japanese :), the changes are in verbatim parts. CC: @fisuda